man/man4/zfs.4

```diff
@@ -1848,6 +1848,17 @@ This setting will not reduce the write size below a single block.
 Capped at a maximum of
 .Sy 32MB .
 .
+.It Sy zfs_recv_best_effort_corrective Ns = Ns Sy 0 Pq int
+.Bl -tag -compact
```

Comment:
-bullet or -enum (rest of zfs.4 uses -enum, so copy that whole line)

Reply:
fixed. Thanks for the reviews